src/utils/addUrlSlashes.ts

```diff
@@ -0,0 +1,12 @@
+// add "//" to the beginning of the url if protocol is missing.
+const addUrlSlashes = (url: string): string => {
+  const protocolRegex = /^((https?:\/\/)|(\/\/))/i;
+  const urlIsMissingProtocol = !protocolRegex.test(url);
+
+  if (urlIsMissingProtocol) {
+    return `//${url}`;
+  }
+  return url;
+};
```

Comment:
Personally I would prefer ```return urlIsMissingProtocol ? `//${url}` : url``` , but this is really just matter of taste

Reply:
updated to use ternary :)